e2e/integration_tests/cpu_forwarding_test.ts

```diff
@@ -17,37 +17,41 @@
 
 import '@tensorflow/tfjs-backend-cpu';
 import '@tensorflow/tfjs-backend-webgl';
+import '@tensorflow/tfjs-backend-webgpu';
 
 import * as tfc from '@tensorflow/tfjs-core';
 // tslint:disable-next-line: no-imports-from-dist
 import {describeWithFlags} from '@tensorflow/tfjs-core/dist/jasmine_util';
 
 import {SMOKE} from './constants';
+import {setBackend} from './test_util';
 
 /**
  *  This file tests cpu forwarding from webgl backend.
  */
 
 describeWithFlags(
-    `${SMOKE} cpu forwarding (webgl->cpu)`, {
-      predicate: testEnv => testEnv.backendName === 'webgl' &&
-          tfc.findBackend('webgl') !== null && tfc.findBackend('cpu') !== null
-    },
+    `${SMOKE} cpu forwarding (webgl->cpu)`,
+    {predicate: testEnv => testEnv.backendName !== 'cpu'},
 
-    () => {
+    (env) => {
       let webglCpuForwardFlagSaved: boolean;
+      let webgpuCpuForwardFlagSaved: boolean;
 
       beforeAll(() => {
         webglCpuForwardFlagSaved = tfc.env().getBool('WEBGL_CPU_FORWARD');
         tfc.env().set('WEBGL_CPU_FORWARD', true);
+        webgpuCpuForwardFlagSaved = tfc.env().getBool('WEBGPU_CPU_FORWARD');
+        tfc.env().set('WEBGPU_CPU_FORWARD', true);
       });
 
       afterAll(() => {
         tfc.env().set('WEBGL_CPU_FORWARD', webglCpuForwardFlagSaved);
+        tfc.env().set('WEBGPU_CPU_FORWARD', webgpuCpuForwardFlagSaved);
       });
 
       it('should work for slice.', async () => {
-        await tfc.setBackend('webgl');
+        await setBackend(env.name);
```

Comment:
Can we move `await setBackend(env.name)` to `beforeAll`? May need to change `beforeAll` to an async func.